routes/search.js

```diff
@@ -4,14 +4,16 @@ const db = require('../dbcon.js');
 const common = require('../common')
 
 // Book object
-function Book(id, title, author, imgUrl, pointcost) {
+function Book(id, title, author, imgUrl, pointcost, isbn, userPoints) {
     this.id = id;
```

Comment:
I think this constructor should just take an object like data and then in the function Book you would have this.id = data.id; etc. The purpose of the constructor is supposed to be to make it easier to create an object, when you are passing in every property of the object separately it almost defeats the purpose as you could just create a new Book() in your code block and manually set the properties and it'd be exactly the same.

Reply:
I have applied this suggestion.